### PR TITLE
Article: backlog priorities

### PR DESCRIPTION
**Pull request by RichardForshaw**

### CODE REVIEW

File: _posts/2018-11-28-backlog-priorities.markdown [hunk 1]
```diff
@@ -0,0 +1,32 @@
+Agile Priorities
```
Comment (Hamo69) [inline _posts/2018-11-28-backlog-priorities.markdown]: have you tested locally? There is no formatting etc, it is just text. If you compare to another post, you will see what I mean. Needs to be in markdown.

File: _posts/2018-11-28-backlog-priorities.markdown [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+================
+
+I have an interest in effective software delivery and I also like watching the videos by Gary Strahan on his 'Development That Pays' channel. When a video popped up on my feed entitled 'From Scrum to Scrumban in 6 steps', I dived in with a mix of inquisitiveness (what is this scrumban thing?) and scepticism (not another agile derivative!). However my inquisitiveness won the day as he went through the 6 steps to transform your scrum-type process.
```
Comment (Hamo69) [inline _posts/2018-11-28-backlog-priorities.markdown]: Could you hyper link to Gary Strahan and his channel? Or add a picture for context?

File: _posts/2018-11-28-backlog-priorities.markdown [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+In the past I have been somewhat obsessed with estimating for many reasons. For one, I still notice that many software developers are still so focussed on the excitement of solving a problem, that they will report in the stand-up meeting that their task is 'nearly done' for several days in a row.
```
Comment (Hamo69) [inline _posts/2018-11-28-backlog-priorities.markdown]: repeated "still" - delete one

File: _posts/2018-11-28-backlog-priorities.markdown [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+I have also convinced myself that anything that is measurable and quantifiable can surely be predicted. However, having delivered several small and large scale software projects, the pessimistic side of me also tends to say 'Software is just too complex - good estimates can't be done'. Then optimisitic side of me counters with 'yes it can! You are probably just expecting too much and measuring the wrong things'.
+
+So imagine my horror when point #5 of Gary's 6-step process was 'Stop Estimating'! What? But how will we schedule the work? How will we budget for it? How will we plan and coordinate launch dates?
```
Comment (Hamo69) [inline _posts/2018-11-28-backlog-priorities.markdown]: pictures!

File: _posts/2018-11-28-backlog-priorities.markdown [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+And here lies the core concept. Applying lean thinking to the question 'when will the product be done so that we can make some money?' transforms it into 'how soon can the product be useful to customers so that they will pay for it?'. And this then looks like an MVP.
+
+What does this mean for the estimate v priorities argument? Determining the features of an MVP is all about priorities. Being fast to market is achieved by deciding *what* features have the most value and what have the least in order to provide value to your customers - i.e. something they will pay for. I suggest this is more important than trying to understand how long it will take to deliver. Estimation still has value, and the company's budget should not be dismissed, but if you are delivering the WRONG features then your budget will also take a big hit. And coincidentally this ties in nicely with point 4 of Gary's video: Start ordering.
+
+So how to prioritize? When it comes to prioritising I usually think about three things:
```
Comment (Hamo69) [inline _posts/2018-11-28-backlog-priorities.markdown]: prioritize.. yank spelling!

File: _posts/2018-11-28-backlog-priorities.markdown [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+ - What features deliver core value
+ - What additional features do you need in order to enable those features
+ - Which features have are the highest risks or carry some auxilliary benefits?
+ 
+Firstly, as we have already discussed, customer value is very important. If you don't provide customers with value then someone else will. Identifying valus should be at the top of your list.
```
Comment (Hamo69) [inline _posts/2018-11-28-backlog-priorities.markdown]: valus - typo

File: _posts/2018-11-28-backlog-priorities.markdown [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+ 
+Secondly, those features will often have other dependent tasks or features: systems will need accounts and logins, notifications will need account profiles or settings, data reporting needs to have data collection. These tasks will need to be done first, but can sometimes be wrapped up in another feature that provides value. For example if you are building a cat-cuteness ranking application, there may be value in building a ranking per-user before a more complex site-wide system.
+ 
+Lastly, don't discount any measurable risk or collateral benefit. If two features are of equal value (however you measure that), I would usually tackle the one with the highest additional benefit first, as futher valuable features may be able to be delivered faster. Beyond that, I also generally try to address the highest-risk items earlier than lowest risk, mainly because as a project marches on those high-risk or high-uncertainty items tend to have a bigger schedule impact the longer they are left.
```
Comment (Hamo69) [inline _posts/2018-11-28-backlog-priorities.markdown]: Love to see a shiny example from Trello / JIRA.

File: _posts/2018-11-28-backlog-priorities.markdown (hunk 1 above)
Comment (RichardForshaw) [reply _posts/2018-11-28-backlog-priorities.markdown]: Yes, and it was rendering for me. Perhaps the `=====` alternative markdown is not recognised everywhere. I've changed it and added some extra headings.

File: _posts/2018-11-28-backlog-priorities.markdown (hunk 2 above)
Comment (RichardForshaw) [reply _posts/2018-11-28-backlog-priorities.markdown]: Added links.

File: _posts/2018-11-28-backlog-priorities.markdown (hunk 3 above)
Comment (RichardForshaw) [reply _posts/2018-11-28-backlog-priorities.markdown]: Done

File: _posts/2018-11-28-backlog-priorities.markdown (hunk 4 above)
Comment (RichardForshaw) [reply _posts/2018-11-28-backlog-priorities.markdown]: Will work out how to do that.

File: _posts/2018-11-28-backlog-priorities.markdown (hunk 5 above)
Comment (RichardForshaw) [reply _posts/2018-11-28-backlog-priorities.markdown]: okee dokee

File: _posts/2018-11-28-backlog-priorities.markdown (hunk 6 above)
Comment (RichardForshaw) [reply _posts/2018-11-28-backlog-priorities.markdown]: fixed

File: _posts/2018-11-28-backlog-priorities.markdown (hunk 7 above)
Comment (RichardForshaw) [reply _posts/2018-11-28-backlog-priorities.markdown]: Hmm... not sure how best to do that.

**Comment by RichardForshaw** [comment]
Addressed comments and smartened up.

**Pull request merged**